### ISSUE TITLE
[🚀DL] Add an option to limit download speeds

### ISSUE THREAD

**Issue by reDpz** [aniyomiorg/aniyomi#157]
## Device information
* Aniyomi version: 0.12.1.1
* Android version: 10
* Device: Huawei p smart 2019

## Steps to reproduce
1. First step: download any anime (not manga)

## Issue/Request
the download speed on my internet is pretty high but the app attempts to download at the same speed with 0 throttling afaik. This wouldn't be an issue if I was on an higher end phone but I'm not so it can cause a lot of lag stuttering and phone getting pretty toasty. This is obv not really an issue for anyone who can afford an actual phone but it would be nice to have a download speed limit. I understand this is a very niché problem and that there might never be a patch or a fix. Btw this stops me from doing anything else in the background although the download is crazy fast like 1 episode in 50 seconds going up to 17mbs (unnecessarily fast for me and causes hardware issues).

TLDR: My phone poopy very slow and app try to go broom broom download too fast for my phone

**Comment by jmir1** [comment]
nice summary XD
i'l remember your suggestion, but no promises if and when i will ever implement this

**Comment by jmir1** [comment]
no need to close it

**Comment by jmir1** [comment]
@reDpz Do you still want this feature?

**Comment by reDpz** [comment]
> @reDpz Do you still want this feature?

no its okay, you dont need to develop it.

**Comment by jmir1** [comment]
Well, one of the devs said they want to try this. I was surprised to learn there are quite a few others in our discord server who want this feature as well.

**Comment by reDpz** [comment]
i mean I'm not saying don't develop it I'm just saying don't develop it for me specifically since I don't need it anymore so therefore it would be a waste of time for you. Also is it okay if i ask if aniyomi will be maintained seeing as development on tachiyomi has stopped?

**Comment by abdallahmehiz** [comment]
Yes the development of aniyomi will be maintained for the foreseeable future.
